htdocs/main.php

```diff
@@ -125,8 +125,7 @@ function tplFromRequest($param)
 
     if (!empty($_REQUEST['sessionID'])) {
         try {
-            $timePoint  =& TimePoint::singleton($_REQUEST['sessionID']);
-            $argstring .= "filter%5Bm.VisitNo%5D=".$timePoint->getVisitNo()."&";
+            $timePoint = TimePoint::singleton($_REQUEST['sessionID']);
```

Comment:
Does this need to change? I'm afraid of conflicts with the NoMainPHP PR if this is unnecessary..

Reply:
looks like this is coming from ur own commit, not sure why it's popping in my PR at all